trellis/wordpress-sites.md

```diff
@@ -88,12 +88,12 @@ For a complete working example of a real-life WordPress site, you can view the c
   * `db_user` - database username (default: `<site name>`)
   * `db_password` - database password (*required*, in `vault.yml`)
   * `db_host` - database hostname (default: `localhost`)
-  
+
 ### Development
 
 * `site_install` - whether to install WordPress or not (default: `true`)
 * `site_title` - WP site title (default: site name)
-* `admin_user` - WP admin user name (default: `admin`)
+* `admin_user` - WP admin user name (default: `admin_email`)
```

Comment:
This could be confusing. Under `db_user` we show the default as another variable like `<site name>`. So maybe `<admin email`> ?